lte/gateway/c/session_manager/SetMessageManagerHandler.cpp

```diff
@@ -190,10 +242,116 @@ void SetMessageManagerHandler::initiate_release_session(
    * print the fatal error and move on
    */
   if (update_success) {
-    MLOG(MINFO)
+    MLOG(MDEBUG)
         << "Successfully released and updated SessionStore of subscriber"
         << imsi;
   }
 }
 
+/* This function called when any specific  session moved to
+ * idle state
+ */
+void SetMessageManagerHandler::pdu_session_inactive(
+    const SetSmNotificationContext& notif,
+    std::function<void(Status, SmContextVoid)> response_callback) {
+  // extract values from proto
+  uint32_t pdu_id  = notif.rat_specific_notification().pdu_session_id();
+  std::string imsi = notif.common_context().sid().id();
+  /* Read the SessionMap from global session_store */
+  SessionSearchCriteria criteria(imsi, IMSI_AND_PDUID, pdu_id);
+  auto session_map = session_store_.read_sessions({imsi});
+  auto session_it  = session_store_.find_session(session_map, criteria);
+  auto& session    = **session_it;
+  if (!session_it) {
+    MLOG(MINFO) << " No session found for IMSI: " << imsi << " pdu id "
+                << pdu_id;
+    Status status(grpc::NOT_FOUND, "Sesion Not found");
+    response_callback(status, SmContextVoid());
+    return;
+  }
+  MLOG(MINFO) << "Received Session Notificaiton from UE with IMSI: " << imsi

```

Comment:
```
[misspell] reported by reviewdog 🐶
"Notificaiton" is a misspelling of "Notification"
```

Reply:
Modified the spelling mistake for "Notification" in the log.